BaseClasses.py

```diff
@@ -424,14 +425,14 @@ def can_shoot_arrows(self, player):
             return self.has('Bow', player) and (self.has('Silver Arrows', player) or self.can_buy_unlimited('Single Arrow', player))
         return self.has('Bow', player)
 
-    def can_get_good_bee(self, player):
-        cave = self.world.get_region('Good Bee Cave', player)
+    def can_get_good_bee(self):
```

Comment:
It looks like the multi-world readyness of this function got accidentally rolled back?

Reply:
Yup, my mistake